ThreadGroup/Controllers/TGRouterAuthViewController.m

```diff
@@ -38,6 +40,9 @@ - (void)resetView {
 #pragma mark - IBActions
 
 - (IBAction)okButtonPressed:(UIButton *)sender {
+    self.view.userInteractionEnabled = NO;
```

Comment:
Disabling userInteraction for the entire view works, however, we will need to cancel any pending request by pressing the "cancel" button(not implemented yet). Disable just the "ok" button instead.